tinyagentos/routes/projects.py

```
@@ -242,6 +242,30 @@ async def list_members(project_id: str, request: Request):
+        await store.set_member_lead(project_id, member_id, body.is_lead)
+    except KeyError as e:
+        return JSONResponse({"error": str(e)}, status_code=404)
+    try:
+        from tinyagentos.projects.a2a import ensure_a2a_channel
+        await ensure_a2a_channel(
+            request.app.state.chat_channels,
+            store,
+            project_id,
+            config=getattr(request.app.state, "config", None),
+        )
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)